src/FsCodec/Codec.fs

```diff
@@ -9,13 +10,13 @@ type Codec =
     // Leaving this helper private until we have a real use case which will e.g. enable us to decide whether to align the signature with the up/down functions
     //   employed in the convention-based Codec
     // (IME, while many systems have some code touching the metadata, it's not something one typically wants to encourage)
-    static member private Create<'Union,'Context>
+    static member private Create<'Union,'Format,'Context>
         (   /// Maps a 'Union to an Event Type Name with UTF-8 arrays representing the <c>Data</c> and <c>Meta</c> together with the correlationId, causationId and timestamp.
```

Comment:
The comments here will need to be updated to indicate that it's UTF-8 agnostic.

Reply:
Thanks for catching...